recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+import os
+import re
+import shutil
+
+from fnmatch import fnmatch
+from conans import ConanFile, CMake, tools
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    version = "9.0.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    license = "BSD license"
+    description = "Visualization Toolkit by Kitware"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    source_subfolder = "source_subfolder"
+    options = {"shared": [True, False], "qt": [True, False], "mpi": [True, False],
+                "fPIC": [True, False], "minimal": [True, False], "ioxml": [True, False],
+                "ioexport": [True, False], "mpi_minimal": [True, False]}
```

Comment:
`mpi` and `mpi_minimal` look fishy.
Can these options be merged?
e.g. `"mp": ["normal", "mini", False]`?

Reply:
Agree and additionally there is `minimal`.
VTK expert please advise

Reply:
Done